boards/pulga/Makefile.features

```diff
@@ -0,0 +1,8 @@
+CPU_MODEL = nrf52840xxaa
+
+include $(RIOTBOARD)/common/nrf52xxxdk/Makefile.features
+
+# Various other features (if any)
+FEATURES_PROVIDED += periph_pwm
+FEATURES_PROVIDED += periph_usbdev
```

Comment:
I didn't see a USB connector on the board page

Reply:
Yep, removed.